streaming.md

```diff
@@ -91,7 +91,8 @@ In particular, the size of the `expectedContentTypes` field (as well as the `out
 
 ### InputFrame
 `InputFrames` represent incoming data to the function. Upon reception of an `InputFrame`, an invoker
-1. SHOULD deserialize the *payload* according to the *contentType* field, which MUST be set and represent a concrete MIME type (*i.e.* not a wildcard type). The data structure that corresponds to the result of de-serialization is runtime-specific and is beyond the scope of this specification. To the best extent though, an invoker SHOULD use data structures that are *idiomatic* to the target runtime and pass them to the function if feasible. As an example, if the incoming `contentType` is `application/json` and `payload` contains the bytes of `{"foo": 5}`, then the invoker may use a *map* or a *dictionary* with key `foo`.
+1. MUST deserialize the *payload* according to the *contentType* field, which MUST be set and represent a concrete MIME type (*i.e.* not a wildcard type). The data structure that corresponds to the result of de-serialization is runtime-specific and is beyond the scope of this specification. To the best extent though, an invoker SHOULD use data structures that are *idiomatic* to the target runtime and pass them to the function if feasible. As an example, if the incoming `contentType` is `application/json` and `payload` contains the bytes of `{"foo": 5}`, then the invoker may use a *map* or a *dictionary* with key `foo`.
```

Comment:
If the *contentType* field is _not_ set, presumably the invoker is free to do anything. Right? Or should it return some gRPC status error code?

Reply:
To be consistent with the request/reply case, I think we should add that if contentType is not set, it should be assumed to be application/octet-stream